prow/cmd/peribolos/main.go

```diff
@@ -23,15 +23,16 @@ import (
 	"os"
 	"strings"
 
+	"github.com/sirupsen/logrus"
```

Comment:
What tool is doing this (moving this import section?)

Reply:
I have a file watcher that runs `goimports` on save. Also, a habit of organizing import in  stdlib, external, internal groups. I think I moved this one manually.